Library/Formula/utf8proc.rb

```diff
@@ -0,0 +1,25 @@
+class Utf8proc < Formula
+  desc "Clean C library for processing UTF-8 Unicode data"
+  homepage "http://julialang.org/utf8proc/"
+  url "https://github.com/JuliaLang/utf8proc/archive/v1.3.1.tar.gz"
+  sha256 "83b60fe21fd8a017b8ad469515873893c8e911a5bef336a427594d398b5688cc"
+
+  def install
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+      #include <string.h>
+      #include <utf8proc.h>
+
+      int main() {
+        const char *version =  utf8proc_version();
+        return strnlen(version, sizeof("1.3.1-dev")) > 0 ? 0 : -1;
+      }
+    EOS
+
+    system env.CC, "test.c", "-I#{include}", "-L#{lib}", "-lpng", "-o", "test"
```

Comment:
I dun goof'd. Teaches me to submit a PR while sleep deprived :weary: The reason is because I based it off the `libpng` formula, and neglected to change it... and test it apparently?

Anyway, it passes now, and I actually tested it >.<

```sh
$ brew test utf8proc
Testing utf8proc
==> Using the sandbox
==> /usr/bin/clang test.c -I/usr/local/Cellar/utf8proc/1.3.1/include -L/usr/local/Cellar/utf8proc/1.3.1/lib -lutf8proc -o test
==> ./test
```